timeless/templates/auth/login.html

```diff
@@ -1,20 +1,23 @@
-{% extends 'base.html' %}
-
-{# @todo #283:30min Change the style of login page. Lets change the#}
-{#  style of a login page to get the UX like demonstrated in a picture below#}
-{#  https://drive.google.com/uc?export=view&id=1oh8cLv-vHTCBmb0RDkHQiyMythC4_M6o#}
-{% cache 600 %}
-{% block header %}
-  <h1>{% block title %}Log In{% endblock %}</h1>
-{% endblock %}
-
-{% block content %}
+<!doctype html>
+<title>{% block title %}{% endblock %} - Timlessis</title>
+<link rel="stylesheet" href="{{ url_for('static', filename='login.css') }}">
```

Comment:
@paulodamaso why does the href have to be dynamic? Can't we just specify the path to the file?